Core/Meta/WellKnownTypes.cs

```diff
@@ -100,6 +102,57 @@ public enum BaseType
         Date,
     }
 
+    public static class BaseTypeHelpers
+    {
+        public static BigInteger MinimumInteger(BaseType type)
```

Comment:
Maybe `MinimumIntegerValue` would make more sense? My first though reading the name was that it would return the min integer type required to hold a given integer value.

Reply:
I think the name is fine as-is, but I added some docs to make it clear what this function does.